repository/src/test/java/uk/gov/hmcts/ccd/definition/store/repository/CaseTypeRepositoryTest.java

```diff
@@ -169,4 +224,54 @@ private void saveCaseTypeClearAndFlushSession(CaseTypeEntity caseType) {
         entityManager.clear();
     }
 
+    @Test
+    public void saveAndGetCaseTypeWithCaseTypeACLData() {
```

Comment:
Can we change this method name as discussed yesterday. This test does alot more and the name is not completely representative of that.